addons/link_tracker/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,39 @@
+---Models in module 'link_tracker'---
+---Fields in module 'link_tracker'---
+link_tracker / link.tracker.click       / campaign_id (many2one)        : NEW relation: utm.campaign, isrelated: related, stored
+link_tracker / link.tracker.click       / click_date (date)             : DEL
+# NOTHING TO DO
```

Comment:
It's good to always say why is `NOTHING TO DO` for future references or for the reviewers, and also leave an empty line for better readability when it's another "block", as we don't have syntax highlighting here in this file:

```suggestion
link_tracker / link.tracker.click       / campaign_id (many2one)        : NEW relation: utm.campaign, isrelated: related, stored
# NOTHING TO DO: Filled by standard update mechanism being a related. Possible improvement: being filled by SQL for performance.

link_tracker / link.tracker.click       / click_date (date)             : DEL
# NOTHING TO DO: This field was not used at all in previous version

```